components/MainInfo/maininfo.module.css

```diff
@@ -0,0 +1 @@
+.container{display:flex;flex-direction:column;justify-content:center;width:100%;min-height:600px}.wrapper{width:100%;display:flex;flex-direction:row;justify-content:center;position:relative;min-height:500px}.image{width:50%;height:400px;margin:0px 20px;align-self:center}.opening-text{display:flex;flex-direction:column;justify-content:center;text-align:center;align-items:center;width:50%;margin:0 40px}.opening-text>p{font-weight:600;font-size:2rem;margin:0}.description{display:flex;flex-direction:column;justify-content:center;width:50%;margin:0px 40px}.description>h2{font-size:1.8rem;text-align:center;margin-bottom:0}.description>p{font-size:1.1rem;color:#595959;line-height:1.5}@media only screen and (max-width: 769px){.container>div:nth-child(1){flex-direction:column-reverse;align-items:center}.container>div:nth-child(2){flex-direction:column;align-items:center}.image,.opening-text,.description{width:80%}.opening-text{margin-bottom:50px}}
```

Comment:
remove compiled css and import directly from scss into MainInfo component

Reply:
When I change the import to be scss instead of the compiled css file, it throws an error saying that the workspace lacks sass. I think sass has to be a dependency in the workspace, am I good to add it as one?

Reply:
Yeah we can go ahead and add it as a dependency